app/controllers/concerns/exception_handler.rb

```diff
@@ -0,0 +1,43 @@
+module ExceptionHandler
+  extend ActiveSupport::Concern
+
+  included do
+    rescue_from StandardError, with: :render_standard_error
+    rescue_from ActionController::ParameterMissing, with: :render_parameter_missing
+    rescue_from ActiveRecord::RecordNotFound, with: :render_record_not_found
+    rescue_from ActiveRecord::RecordInvalid, with: :render_record_invalid
+  end
+
+  def render_errors(error_messages, status)
```

Comment:
instead of casing by if its an array or not, I would create two different methods. One for `render_error` and other for `render_errors`, Thoughts?